clang/lib/CIR/Dialect/Transforms/TargetLowering/Targets/X86.cpp

```diff
@@ -0,0 +1,31 @@
+#include "ABIInfo.h"
+#include "LowerModule.h"
+#include "LowerTypes.h"
+#include "MissingFeature.h"
+#include "TargetInfo.h"
+#include <memory>
+
+namespace mlir {
+namespace cir {
+
+class X86_64ABIInfo : public ABIInfo {
```

Comment:
Similar here, please already unify these with the stuff in lowering prepare, and the existing Aarch64 stuff.

Reply:
I intend to migrate and unify them, but I would rather do it as a second step. One of the core ideas of this library is to leverage codegen parity to facilitate development. The existing `LoweringPrepareAArch64CXXABI` seems to have its own logic instead of copying the original codegen.

Reply:
I'll do the suggested approach: move it and add `FIXME`s to patch it later.